src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
+@import '../utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-columns;
+  max-width: ($day-size + $gap-columns) * $columns;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid black;
+
+    @for $number from 1 through 31 {
+      &:nth-child(#{$number})::before {
+        content: '#{$number}';
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: $bg-hover;
+      transform: translateY(-20px);
+      animation-name: transform;
+      animation-duration: 500ms;
+    }
```

Comment:
The 'animation-name' and 'animation-duration' properties are specified directly within the '&:hover' selector. It's generally a good practice to define keyframes for the animation outside of the selector and then reference it within '&:hover'. This improves maintainability and separation of concerns.